### ISSUE TITLE
[bug] Double-quotes on CssJs.unsafe for font-family that include quotes?

### ISSUE THREAD

**Issue by AlexMoutonNoble** [davesnx/styled-ppx#342]
Hi David!
Our team has been using quotes for font family strings that have whitespace:

```
font-family: "Inter Semi Bold";
```

This seems like it should be valid, maybe single quotes?, but produces:

```
CssJs.unsafe("fontFamily", ""Inter Semi Bold""),
```

I have removed the quotes for now but maybe something to be handled here?

Thanks
Alex

[edit: this on 0.36.2]

**Comment by davesnx** [comment]
Hey @AlexMoutonNoble, that's a bug for sure. It should generate `CssJs.unsafe("fontFamily", {js|"Inter Semi Bold"|js})`.

**Comment by davesnx** [comment]
Hey @AlexMoutonNoble, what's the annotation you use for font family in this case? (`cx`, `styled`, `global`? and probably you are using ReScript, right?

**Comment by davesnx** [comment]
Fixed in 7f0594d2b8dcc2c86753fa699bd32a652bcdfe22
